src/Microsoft.ML.Transforms/CategoricalCatalog.cs

```diff
@@ -43,14 +50,19 @@ public static class CategoricalCatalog
                 new[] { new OneHotEncodingEstimator.ColumnOptions(outputColumnName, inputColumnName, outputKind, maximumNumberOfKeys, keyOrdinality) }, keyData);
 
         /// <summary>
-        /// Convert text columns into one-hot encoded vectors.
+        /// Create a <see cref="OneHotEncodingEstimator"/>, which converts one or more input text columns specified in <paramref name="columns"/>
+        /// into as many columns of one-hot encoded vectors.
```

Comment:
… vectors.  All of the columns will be processed in a single pass through data.   -- we need to explain that it is more efficient to specify one transform with many columns vs many transforms with a single column, perhaps in remarks?